utilities/src/bytes.rs

```diff
@@ -80,7 +80,7 @@ pub trait FromBytes {
     }
 }
 
-pub struct ToBytesSerializer<T: ToBytes>(String, Option<usize>, PhantomData<T>);
+pub struct ToBytesSerializer<T: ToBytes>(PhantomData<T>);
```

Comment:
Can you maybe share intuition or the clippy lint which led to these suggestions? The change ot `FromBytesVisitor` makes sense but for `ToBytesSerializer` and `FromBytesDeserializer` I don't see why we can remove those elements "for free" and why we put them there in the first place.

Reply:
`clippy` just stated that they are never used